constraint-solver/src/solver/base.rs

```diff
@@ -562,13 +562,17 @@ where
     }
 }
 
+/// Returns true if the constraint is equivalent to `X = Y` for
+/// some variables `X` and `Y`.
+/// In this case, returns the "larger" variable and the result
```

Comment:
Larger because it's the second one in the btreemap?

Reply:
Yes